zebra-utils/zcash-rpc-diff

```diff
@@ -27,7 +27,9 @@ fi
 ZEBRAD_RPC_PORT=$1
 shift
 
-ZCASH_RPC_TMP_DIR=$(mktemp -d)
+# Use an easily identified temp directory name,
+# but fall back to a generic name if `mktemp` is outdated.
```

Comment:
How can mktemp be outdated and what does this really means ?

Reply:
Older versions of the `mktemp` command might not support the `--suffix` argument.

```suggestion
# but fall back to the default temp name if `mktemp` does not understand `--suffix`.
```